api/src/pcapi/routes/backoffice_v3/individual_bookings.py

```diff
@@ -245,3 +246,77 @@ def mark_booking_as_cancelled(booking_id: int) -> utils.BackofficeResponse:
         flash(f"La réservation {booking.token} a été annulée", "success")
 
     return _redirect_after_individual_booking_action()
+
+
+@individual_bookings_blueprint.route("/batch-validate", methods=["GET"])
+def get_batch_validate_indivual_bookings_form() -> utils.BackofficeResponse:
+    form = booking_forms.BatchActionOnBooking()
```

Comment:
```suggestion
    form = booking_forms.BatchForm()
```
ainsi que sur les usages suivants de `BatchActionOnBooking`